tests/integration/targets/vyos_firewall_global/vars/pre-v1_4.yaml

```diff
@@ -59,6 +59,62 @@ merged:
         connection_type: invalid
     twa_hazards_protection: true
     validation: strict
+  config:
+    validation: strict
+    config_trap: true
+    log_martians: true
+    syn_cookies: true
+    twa_hazards_protection: true
+    ping:
+      all: true
+      broadcast: true
+    state_policy:
+      - connection_type: established
+        action: accept
+        log: true
+      - connection_type: invalid
+        action: reject
+    route_redirects:
+      - afi: ipv4
+        ip_src_route: true
+        icmp_redirects:
+          send: true
+          receive: false
+    group:
+      address_group:
+        - name: MGMT-HOSTS
+          description: This group has the Management hosts address list
+          members:
+            - address: 192.0.1.1
+            - address: 192.0.1.3
+            - address: 192.0.1.5
+      network_group:
+        - name: MGMT
+          description: This group has the Management network addresses
+          members:
+            - address: 192.0.1.0/24
+
+populate_config:
```

Comment:
can we change this to `populate_commands`?  This is a list of commands to send, not a config record.